src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts

```diff
@@ -35,7 +36,7 @@ describe('OutcomeBehaviourMapProvider', () => {
     });
 
     outcomeBehaviourMapProvider = TestBed.get(OutcomeBehaviourMapProvider);
-    outcomeBehaviourMapProvider.setBehaviourMap(behaviourMap);
+    outcomeBehaviourMapProvider.setBehaviourMap(simpleBehaviourMap);
```

Comment:
Can't see where we're using this `setBehaviourMap`? We seem to be resetting it to use `behaviourMap` in all the tests?